dompurify/dompurify.d.ts

```diff
@@ -0,0 +1,15 @@
+// Type definitions for DOM Purify
+// Project: https://github.com/cure53/DOMPurify
+// Definitions by: Dave Taylor <http://davetayls.me>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+
+interface IDOMPurify {
+  sanitize(s:string):string;
+  addHook(hook:string, cb:(currentNode:Element, data:any, config:any) => Element):void;
+}
+
+declare var DOMPurify:IDOMPurify;
```

Comment:
only issue with this is that when you include this in the browser with a script tag it creates a global `DOMPurify` namespace on the window object which is why I included this statement